README.md

```diff
@@ -19,24 +19,54 @@ The usage is pretty similar to the handling HTTP request, but instead of routes
 we are using message types.
 
 ```go
-// Define the OpenFlow handler for hello messages.
-openflow.HandleFunc(openflow.TypeHello, func(rw openflow.ResponseWriter, r *openflow.Request) {
+package main
+
+import (
+    of "github.com/netrack/openflow"
```

Comment:
It still seems to me that ```of``` looks a bit overloaded, so if possible, it would be better to keep in the documentation the renamed imports.